app/extensions/restManager/RestManager.py

```diff
@@ -112,19 +111,21 @@ def _endpoint_fmtstr(self, tag, target='default'):
         return endpoint_fmtstr
 
     def _endpoint_tag_fmtstr(self, tag):
-        endpoint = self.ENDPOINTS
+        endpoints = self.ENDPOINTS
```

Comment:
This is a bug fix. 
If the endpoint wasn't found the existing code returned the json for all of the endpoints, it now returns None